src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -36,6 +36,7 @@
         :on-blur               #(reset! input-focused? false)
         :on-scan               (fn []
                                  (rn/dismiss-keyboard!)
+                                 (rf/dispatch [:wallet/clean-scanned-address])
```

Comment:
What if the user accidentally taps the scan button after scanning and dismisses the screen? Are we sure we want to always clean the scanned address when opening this screen?

Reply:
Actually it might make sense becuase the scan button is hidden in favour of the clear button when there is value in the input...